index.html

```diff
@@ -6,10 +6,176 @@
-    <title>RG Conference-2022</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,200;0,400;1,600&display=swap" rel="stylesheet">
+    <title>CC Summit-2022</title>
 </head>
 <body>
-    <h1>RG Conference - 2022</h1>
+    <div class="header-page">
+        <div class="mobile-menu container-fluid pt-5 gap-5 bg-dark">
+            <ul class="menu gap-1 fs-5 p-5">
+                <li><a href="./index.html">Home</a></li>
+                <li><a href="./about.html">About</a></li>
+                <li><a href="#main-program">Program</a></li>
+                <li><a href="#main-program">Join</a></li>
+                <li><a href="#partner">Sponsor</a></li>
+                <li><a href="">News</a></li>
+                <li><a href="./about.html">Contact</a></li>
+            </ul>
+        </div>
+        <div class="desk-header-menu justify-content-around">
+            <a href="./index.html"><img class="header-logo" src="./image/headerLogo.png" alt="logo"></a>
+            <div class="desktop-menu d-flex">
+                <div class="menu-desktop d-flex gap-5 p-4">
+                    <a href="./index.html">Home</a>
+                    <a href="./about.html">About</a>
+                    <a href="#main-program">Program</a>
+                    <a href="#main-program">Join</a>
+                    <a href="#partner">Sponsor</a>
+                    <a href="">News</a>
+                    <a href="./about.html">Contact</a>
+                </div>
+            </div>
+        </div>
+        <div class="top container-fluid pt-4 p-4">
+            <header class="header position-fixed">
+                <div class="bar" id="bar1"></div>
+                <div class="bar" id="bar2"></div>
+                <div class="bar" id="bar3"></div>
+            </header>
+            <main class="main">
+                <div class="main-content pt-5">
+                    <h2>"Hello Sharing World"</h2>
+                    <H1>CARBON CAPTURE SUMMIT - 2022</H1>
+                </div>
+                
+                <p class="border border-3 border-white">Welcome to the Carbon Capture Summit 2022, where industry leaders will meet in Amsterdam to explore next-generation technologies for carbon capture, utilisation, storage, and transportation to meet net-zero emissions targets.</p>
+                <h2 class="text-dark">2022.11.24(THR) ~ 25(FRI)</h2>
+                
+                <h5>National Museum, Van Der Valk, Amsterdam, Netherlands</h5>
+            </main>
+        </div>
+        <div class="" id="main-program">
+            <h3 class="text-white text-center pt-4">Main Program</h3>
+            <h3 class="text-danger text-center">____</h3>
+            <div class="mpcontents">
```

Comment:
- [ ] Please ensure the horizontal line in your program heading has a little space from the program cards. Right now it does not conform to the design parameters of behance.
![image](https://user-images.githubusercontent.com/90556221/183043920-ca98fbf5-a17e-4c18-a8f7-debe98eee3dc.png)

Reply:
Made the necessary changes.